style/fftools-logo.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128">
+    <!-- base64-encoded 128x128 png image of firefly logo -->
```

Comment:
@gpdf yes for that reminder, I've put a comment in svg file itself. I can change the text for more clarity?

Reply:
Github shows "rich diff" by default, you'll need to toggle to "source diff" by using toggle buttons on top right of this file.